### ISSUE TITLE
Hypergolic_UpperWhite not working

### ISSUE THREAD

**Issue by lukecologne** [KSP-RO/RealPlume#53]
The new plume Hypergolic_UpperWhite is not working when ReStock is not installed. I looked around a bit in the configs, and found this:`modelName = ReStock/FX/restock-fx-wolfhound-core`
So, it seems like the FX model is being taken right out of ReStock, instead of RealPlume. I did a quick fix by replacing it with this: `modelName = RealPlume/Nertea_FX_New/ReStock/restock-fx-wolfhound-core`
Now I got another issue, the plume looks now like this:
![screenshot57](https://user-images.githubusercontent.com/39596827/64475995-d5d8f100-d189-11e9-8e2c-52501e8125b4.png)

I can confirm that this "blockification" only affects restock-fx-wolfhound-core, and not restock-fx-wolfhound-plume.

**Comment by zorg2044** [comment]
Thanks for reporting this. I think I know what happened with the blockification, maybe a missing texture.

**Comment by lukecologne** [comment]
Should I open a PR because of the wrong model path? I didn't find any other occurrence of this, but I also didn't go through all plumes.

**Comment by zorg2044** [comment]
No thats ok, I need to check on the texture issue first and check the paths on the others. I had originally created some of these to include within reStock itself before ending up upstreaming here instead. and others are brought in from stuff I did for BluedogDB so I had to change the paths. Will double check again.

**Comment by zorg2044** [comment]
Fixed in the latest dev branch commit https://github.com/KSP-RO/RealPlume/commit/f6f53a7bda9f5cb1ce9c940dca8a68f0d75c174e

I probably won't make a release for a couple of days as theres a lot that changed in both RealPlume and RealPlume-Stock and I want to wait and see if any other issues crop up.

**Comment by lukecologne** [comment]
Great, thank you for the quick fix, I will try to update the RO configs to the new plumes where possible, so I will be looking through many of them. I will report back any issues I find.

**Comment by lukecologne** [comment]
Since the issue has been resolved, I think it's ok to close the thread now.

**Comment by zorg2044** [comment]
@lukecologne I actually came across a couple of missing parameters in hypergolic_upper_white (global emissionMult) and while fixing this I realised I could use a better flare effect. So any configs that have used this would need to be adjusted to account for the differently sized flare effect. Really sorry about that! But it should look better. Hopefully you havent applied it to too many engines yet 0_0. 

A reduction in size to 40% of the original scale value has worked for me, had to adjust position slightly for some engines.

Don't worry, I don't plan to make a habit of breaking changes like this, but I dont think anyone has done configs with this effect apart from both of us. 

Commit in Dev branch https://github.com/KSP-RO/RealPlume/commit/d9188113cbfe5a295d81b725490577f3079adc55
